admin/src/js/controllers/main.js

```diff
@@ -21,16 +22,20 @@ angular.module('controllers').controller('MainCtrl',
           }
           return response;
         });
-    };   
-    
+    };
+
     $translate.use('en');
     $scope.authorized = false;
     $scope.navbarCollapsed = true;
-    Auth.any([['can_configure'], ['can_view_outgoing_messages'], ['can_export_all']])
-      .then(isAuthorized => {
-        $scope.authorized = isAuthorized;
+    $q
+      .all([
+        Auth.online(true),
```

Comment:
Why are we limiting this to online users only? The app won't function offline, but offline users with the right permissions should be able to use the admin app while they have an internet connection I think.

Reply:
Yes. We got this for free by offline users not being allowed to see the admin ddoc before. 
https://github.com/medic/cht-core/pull/4892 https://github.com/medic/cht-core/issues/4786#issuecomment-428335797

Reply:
Ok, as long as it's backwards compatible! Thanks for refreshing my memory...